### DISCUSSION TITLE
Add Greencube?

### DISCUSSION THREAD

**Discussion by LarsThunberg** [magicbug/Cloudlog#1753] [General]
Hi, is it possible to add the Greencube satellite?
TNX!
/Lars SM0TGU

**Comment by magicbug** [comment]
Its in the dev branch, will be in the master sometime tomorrow when I do a merge up

**Reply by LarsThunberg** [reply]
Thank you Peter!

**Reply by LarsThunberg** [reply]
Strange, I can not select Greencube after update to v2.1, but I can see that it is present in file satellite_data.json

**Reply by magicbug** [reply]
You might need to CTRL F5 the browser window

**Reply by LarsThunberg** [reply]
Oh!! That cache, it hit me every time :)  Yes it worked now, thank you.

**Reply by magicbug** [reply]
Cache is great for speed but catches you out sometimes

**Reply by LarsThunberg** [reply]
Follow up: if Greencube will be added to LoTW with name IO-117, should the name in Cloudlog be changed?

**Comment by magicbug** [comment]
If you check the dev branch it's already there, holding back merge for Adif spec update that's due to add some pota fields then there will be a stable release.

**Reply by LarsThunberg** [reply]
Thank you Peter for fast respons!
